gestionareOrase/gestionareOraseV5.py

```diff
@@ -0,0 +1,153 @@
+import os
+version = '0.4.005'
+
+class Oras():
+    def __init__(self, ordX='', ordY='', nume='', judet='', auto='', populatie='', regiune=''):
+        self.ordX = ordX
+        self.ordY = ordY
+        self.nume = nume
+        self.judet = judet
+        self.auto = auto
+        self.populatie = populatie
+        self.regiune = regiune
+        
+    def __str__(self):
+        return self.ordX  +  self.ordY  +  self.nume  +  self.judet  +  self.auto  +  self.populatie  +   self.regiune
+        
+    def getOrdX(self):
+        return self.ordX
+    
+    def getOrdY(self):
+        return self.ordY
+    
+    def getNume(self):
+        return self.nume
+    
+    def getJudet(self):
+        return self.judet
+    
+    def getAuto(self):
+        return self.auto
+    
+    def getPopulatie(self):
+        return self.populatie
+    
+    def getRegiune(self):
+        return self.regiune
+
+def loadFile(file):
+    f = open(file,'r')
+    arr = []
+    for line in f:
+        
+        latitudine = line[0:line.index(',')]  # aici obtin literele de la poz [0] pana inainte de pozitia primei , gasite
+        line = line[line.index(',')+1:] # mai sus am obtinut ce doream deci tre sa sterg totuil pana la prima , inclisv
+        
+        
+        longitudine = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+        
+        
+        oras = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+       
+        
+        judet = line[0:line.index(',')] 
+        line = line[line.index(',')+1:]
+       
+        
+        judetAuto = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+       
+        
+        populatie = line[0:line.index(',')]
+        line = line[line.index(',')+1:]
+        
+        regiune = line[0:]
+       
+        arr.append(Oras(latitudine, longitudine, oras, judet, judetAuto, populatie, regiune))
+                   
+    f.close()
+    
+    return arr
+
+def menu():
+    print("1. RO")
+    
+        
+OP_EXIT = '10'
+
+op = 0
+
+while op != OP_EXIT:
+    
+    op = input("Introduceti  optiunea dorita pentru a continua in limba romana RO sau limba engleza EN: ")
+    
+    print("RO")
+    print("EN")
+    print ("10. EXIT")
+   
+    
+    
+    
+    
+    if op == "RO":
+        
+        
+        
+        OP_EXIT1 = "10"
+        while op != OP_EXIT1:
+            os.system('cls')
+        
+            menuOption01 = "1.Citire fisier"

```

Comment:
ca sa nu faci if op == "RO" valorile din menuOption01 le iei fie din fisierul de romana fie din fisierul de engleza in functie de limba selectata